src/details/ArborX_DetailsTreeConstruction.hpp

```diff
@@ -133,12 +133,12 @@ inline void initializeSingleLeafNode(ExecutionSpace const &space,
 {
   using Access = AccessTraits<Primitives, PrimitivesTag>;
 
-  ARBORX_ASSERT(static_assert<int>(leaf_nodes.extent(0)) == 1);
-  ARBORX_ASSERT(static_assert<int>(Access::size(primitives)) == 1);
+  ARBORX_ASSERT(leaf_nodes.extent(0) == 1);
+  ARBORX_ASSERT(Access::size(primitives) == 1);
```

Comment:
meant to be a cast

Reply:
Feel free to force push erasing my commit. I just wanted to get something built.